test/e2e/testdata/even-odd.yaml

```diff
@@ -14,13 +14,19 @@ spec:
           image: quay.io/numaio/numaflow-go/map-even-odd
     - name: even-sink
       sink:
-        log: {}
+        udsink:
+          container:
+            image: quay.io/numaio/numaflow-go/sink-redis:test-v3
```

Comment:
This is my test image, will switch to `quay.io/numaio/numaflow-sink/redis-e2e-test-sink:latest` once PR https://github.com/numaproj/numaflow-sinks/pull/11 gets merged.

Reply:
It is okay to point to `-v3`.

Reply:
I didn't notice that it is pointing to the wrong quay path. It would have been fine if it was pointing to `quay.io/numaio/numaflow-sink/redis-e2e-test-sink:v3`

Reply:
Agree. By publishing a v3, we can ensure a smooth transfer to new version without breaking other developer's e2e tests. Will try this approach next time.